### ISSUE TITLE
Improve code structure

### ISSUE THREAD

**Issue by vitalii-bulyzhyn** [diffgram/diffgram#523]
As soon as text interface will be finished, Diffgram will have 3 separate interfaces (and in future we will have more I guess):
- Image/Video
- 3D
- Text

The problem here is that some of the components and classes, that are used everywhere, are located in annotation directory (which contains all the code for Image/Video interface). In future, it will have more sense to have move "command" folder and annotation_core_command_manager.js outside of annotation just to make it clear that they are used for all the interfaces and not only for Image/Video

**Comment by PJEstrada** [comment]
Yes I think this is a great Idea. For better readibility we can create subfolders on annotation/ folder like annotation/image , annotation/text, etc and move the files that are global to all to just annotation/
